Template for Trainings page

sudo-make-coffee opened the pull request:
Added new template for Trainings page

sudo-make-coffee committed:
Trying to add new pages

sudo-make-coffee committed:
Trying to add new pages

sudo-make-coffee committed:
Current state

sudo-make-coffee committed:
Adding assets for new pages

granicz committed:
We punt on the home page just yet, and make Consulting the home page.

sudo-make-coffee committed:
Template for new Trainings page

Jooseppi12 commented on src/Hosted/.vs/VSWorkspaceState.json:
```diff
@@ -0,0 +1,6 @@
+{
+  "ExpandedNodes": [
+    ""
```
@erolaliyev Can you push a commit which changes the .gitignore file by adding the `src/Hosted/.vs` path to it and remove the files that were added to the repo from that folder?

sudo-make-coffee committed:
Removed .vs folder files

sudo-make-coffee committed:
Added new application page for single job opening #10

Jooseppi12 committed:
#17 - Minifying css, converting custom.css to scss, setting up grunt pipeline for scss conversion and css minification, adding option for release vs debug mode build with configuring to use the proper variant, remove unused bulma scss files from the repo

sudo-make-coffee committed:
Added content for careers page #10

Jooseppi12 committed:
Extend tooling, create vscode workspace with some predefined settings/tasks

Jooseppi12 committed:
#17 - Use minified css in html templates, make sure the font-awesome iconset is included in every page, removing iconsmind icon set wherever possible, enable sourcemap

Jooseppi12 commented on src/Hosted/openPosition.html:
```diff
@@ -0,0 +1,165 @@
+                  <div class="row">
+                    <div class="col-lg-4">
+                        <div class="icon-box icon-box-left">
+                            <i class="im-air-balloon"></i>
```
@erolaliyev We should not use the iconsmind icon set, we should use icons from the Font Awesome set to achieve consistency. (We need Font Awesome for the social icons). At the moment if I'm not mistaken the following pages still have iconsmind icons:

- openPosition.html
- careers.html
- trainings.html

Jooseppi12 committed:
#17 - Use hexadecimal color codes consistently in custom.scss

Jooseppi12 committed:
#7 Update scripts for more granular handling of processes, remove npm related script execution from f# projects

sudo-make-coffee committed:
Added content for Trainings page #19

sudo-make-coffee committed:
Added content for Careers and Open Position pages #10

Jooseppi12 committed:
Remove generated custom.css file, Fix infinite loop in grunt-watch, Removing extra comment close tag from prettify.css

granicz committed:
RSS/Atom feed links and styling on user blog pages.
Removed the `Jost` font artifacts, we are now using Roboto.
Minor CSS tweaks, including menu-wide on all pages.
Added our local offices to Contact Us.

sudo-make-coffee committed:
#10 Changed content for Open Position page

sudo-make-coffee committed:
#19 Changed images for Trainings page

sudo-make-coffee committed:
#10 Changed content for Jobs and Open Position pages

sudo-make-coffee committed:
Change icons to Font Awesome in Careers, Open Position and Trainings pages

sudo-make-coffee committed:
Trying to replace images

granicz committed:
There are dark and light menubars - depending on the banners/images they are on. Use these css classes.

sudo-make-coffee committed:
#10 Optimized images for Open Position page

sudo-make-coffee committed:
#10 Optimized images for Jobs page and added FAQ section

sudo-make-coffee committed:
#19 Optimization for images on Trainings page

sudo-make-coffee committed:
#19 Image optimization for Trainings page

sudo-make-coffee committed:
#9 Optimizing images for Consulting page

Jooseppi12 committed:
Change open source menu style

sudo-make-coffee committed:
#10 Fixed typo on Open Position page

sudo-make-coffee committed:
#9 Optimizing images for Consulting page

Jooseppi12 committed:
Remove Bulma Drawer, Change header styling

Jooseppi12 committed:
Change header stylings here and there

Jooseppi12 committed:
#8 - Research page content WIP

Jooseppi12 committed:
Make Careers header non-sticky, Add hover information for Contact page gmaps

Jooseppi12 committed:
Fixes after rebase

Jooseppi12 committed:
Add extra sections

Jooseppi12 committed:
Update install.ps1

Jooseppi12 committed:
Fix header issue (caused by footer), fix cp command

granicz committed:
Added article keys to the debug screen.

sudo-make-coffee committed:
Add date and author section to individual blog posts

sudo-make-coffee committed:
#10 Created new page for intern position and added extra section to Careers page

Jooseppi12 committed:
Remove node-sass from dependencies, as it is not required

Jooseppi12 committed:
Fix vscode tasks, update node-sass to the latest package

Jooseppi12 committed:
Update outdated package, remove unused file from repository

mrtank committed:
#19 add training links

mrtank committed:
#19 - Courses are scraped from trainings folder

Jooseppi12 committed:
#19 Change relative to absolute links

sudo-make-coffee committed:
Implemented jobs folder structure for new open positions in the future

mrtank committed:
#19 - Tutorialing instead of courses.

mrtank committed:
#19 - Add Call to Action email actions. Fix folder scraping issues.

Jooseppi12 committed:
#6 - Update build scripts to use normal verbosity

granicz committed:
Added a master sidebar to the job pages that describes the application process.

mrtank committed:
#9 #19 hover zoom. Content for consulting. Fix url encode for subject.

sudo-make-coffee committed:
#10 Added FAQ for individual open job position pages

mrtank committed:
#19 #9 - Links to the main IntelliFactory tools. li links into snippets.

Jooseppi12 committed:
Add Dockerfile

Jooseppi12 committed:
Add helper scripts for Docker

granicz committed:
Moved the blog page template from index.html to blogpost.html - might need to clean this new file up a bit.

mrtank committed:
#9 - full path for assets. Try to use progress.js. Use themekit zoom in solution. Add repos + full-stack apps counters. Add links to contact page on consulting.

sudo-make-coffee committed:
#9 Fixed the number counter

granicz committed:
Fix #26 - Add common metatags to blog pages

mrtank committed:
#9 h2 fix. Faster counter.

mrtank committed:
#19 add hints for typo in url.

Jooseppi12 committed:
#27 - Add metatags

granicz committed:
Fixed date format and tweaked style on the blog pages.

Jooseppi12 committed:
#28 - Categoires page

Jooseppi12 committed:
Added time to read section

Jooseppi12 committed:
Fix issue with blog tags visual

granicz committed:
Updated to WS 4.7.x and FCore 5, and markdig/yamldotnet to their latest.
